src/servers/src/http/influxdb.rs

```diff
@@ -39,13 +44,29 @@ pub async fn influxdb_health() -> Result<impl IntoResponse> {
     Ok(StatusCode::OK)
 }
 
+fn is_gzip(headers: &HeaderMap) -> Result<bool> {
+    match headers
+        .get("Content-Encoding")
+        .map(|val| val.to_str().context(InvisibleASCIISnafu))
```

Comment:
We use these snafus to indicate invalid arguments.

But they are currently all mapped to `InvalidAuthHeader`.

Maybe we need to add more specific snafus for these cases.

Reply:
+1